frontend/index.html

```
@@ -58,6 +58,10 @@ <h1 class="brand">
               <span class="picker-label">ルートを選ぶ (GPX / FIT)</span>
               <input id="gpx-file" type="file" accept=".gpx,.fit,application/gpx+xml,text/xml,application/xml,application/octet-stream" />
             </label>
+            <form class="rwg-form" id="rwg-form" autocomplete="off">
+              <input id="rwg-url" type="text" placeholder="または RWG URL / ID" inputmode="url" autocomplete="off" />
```

HTMLHint (1.9.2)

[warning] No matching [ label ] tag found.

(input-requires-label)